Unterminated inline tags in the MathHelper class

The documentation for the `nextInt` and `nextBetween` method includes an unterminated `{@return}` inline tag:

```
./net/minecraft/util/math/MathHelper.java:414: error: unterminated inline tag
   * {@return a random, uniformly distributed integer value in {@code
              ^
./net/minecraft/util/math/MathHelper.java:1157: error: unterminated inline tag
   * {@return a random, uniformly distributed integer value in {@code
              ^
2 errors
```

--- a/mappings/net/minecraft/util/math/MathHelper.mapping
+++ b/mappings/net/minecraft/util/math/MathHelper.mapping
@@ -167,7 +167,7 @@ CLASS net/minecraft/class_3532 net/minecraft/util/math/MathHelper
 	METHOD method_15394 randomUuid ()Ljava/util/UUID;
 	METHOD method_15395 nextInt (Ljava/util/Random;II)I
 		COMMENT {@return a random, uniformly distributed integer value in {@code
-		COMMENT [min, max]} If the range is empty (i.e. {@code max < min}), it
+		COMMENT [min, max]}} If the range is empty (i.e. {@code max < min}), it
 		COMMENT returns {@code min}.
 		ARG 0 random
 		ARG 1 min
@@ -268,7 +268,7 @@ CLASS net/minecraft/class_3532 net/minecraft/util/math/MathHelper
 		ARG 2 max
 	METHOD method_32751 nextBetween (Ljava/util/Random;II)I
 		COMMENT {@return a random, uniformly distributed integer value in {@code
-		COMMENT [min, max]}
+		COMMENT [min, max]}}
 		COMMENT
 		COMMENT @throws IllegalArgumentException if the range is empty (i.e. {@code
 		COMMENT max < min})

